src/model/EnergyManagementSystemActuator.hpp

```diff
@@ -85,13 +94,17 @@ namespace model {
     /** @name Getters */
     //@{
 
-    boost::optional<ModelObject> actuatedComponent() const;
+    boost::optional<ModelObject> actuatedComponent() const;  // TODO: this should NOT be an optional
```

Comment:
I didn't want to break API, but this should never have been an optional. Though I suppose the idea is that there's a chance the object it referenced would be deleted, right?

Reply:
Thats right.  what happens if that object gets removed?

Reply:
You just don't translate it during FT. That can mean having the FT calling an Impl function or in a try catch
In an ideal world, removing à modelobject would mean removing any ems actuator attached, but that's probably a major slow down and we don't want to penalize anyone not using ems